labelmerge/workflow/scripts/label_split.py

```diff
@@ -17,30 +18,49 @@ def load_atlas(atlas_path):
     return data, header, affine
 
 
-def load_metadata(atlas_path):
+def load_metadata(atlas_path, bids_dir, smk_wildcards):
     """Load metadata associated with atlas"""
-    # Find and read associated metadata file with atlas
-    try:
-        tsv_path = atlas_path.replace("nii.gz", "tsv")
-        metadata = pd.read_csv(tsv_path, sep="\t")
-    except:
-        raise FileNotFoundError(f"{tsv_path} file not found")
-
-    # Create new column storing description for filename
+    # Walk backwards to find dseg until bids_dir is hit
+    tsv_file = None
+    cur_dir = atlas_path.parent
+    while cur_dir != bids_dir and tsv_file == None:
```

Comment:
Won't this prevent us from looking in the roots bid directory?

Reply:
Yes and actually thought I had pushed some code to actually fix this (e.g. while cur != bids_dir.parent) I will take a closer look next week.

Reply:
This is now fixed in 1ddf2f5